metron-stellar/stellar-common/src/test/java/org/apache/metron/stellar/dsl/functions/DateFunctionsTest.java

```diff
@@ -208,6 +240,12 @@ public void testDayOfYear() {
     assertEquals(238, result);
   }
 
+  @Test
+  public void testDayOfYearSeconds() {
+    Object result = run("DAY_OF_YEAR(epoch_seconds)");
+    assertEquals(238, result);
+  }
+
```

Comment:
Suggest adding tests for an epoch time at or before Sunday, September 9, 2001 1:46:39 AM.